### ISSUE TITLE
Compiler module missing from Python 2.7 wheels

### ISSUE THREAD

**Issue by michaelbausor** [google/protobuf#2895]
The `google.protobuf.compiler` module is missing in the Python 2.7 wheels for releases 3.2.0 and 3.0.0 on PyPI:
https://pypi.python.org/pypi/protobuf/3.2.0
https://pypi.python.org/pypi/protobuf/3.0.0

This only seems to affect the wheels for Python 2.7, which are only available on PyPI for those two releases. The module is still present in the source distribution and Python 3.x wheels.

To reproduce missing module error:

`$ pip install protobuf==3.2.0`
`$ python`
Python 2.7.6 (default, Oct 26 2016, 20:30:19) 
[GCC 4.8.4] on linux2
Type "help", "copyright", "credits" or "license" for more information.
`>>> import google.protobuf`
`>>> import google.protobuf.compiler`
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named compiler

**Comment by michaelbausor** [comment]
Is there any plan to create a point release of `protobuf` so that the most recent version does not have this issue?

**Comment by lexrj** [comment]
This issue had stopped occurring with release 3.3.0, but has started reoccurring with 3.4.0:
```
Traceback (most recent call last):
  File "/usr/local/bin/protoc-gen-sidecar", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/vagrant/src/pysidecar/bin/protoc-gen-sidecar", line 6, in <module>
    from google.protobuf.compiler import plugin_pb2 as plugin
ImportError: No module named compiler
```

Uninstalling 3.4.0 and reinstalling 3.3.0 causes the issue to stop occurring.

**Comment by drielsma** [comment]
+1 this regressed for me as well (macOS, installing protobuf via brew)

**Comment by garrettjonesgoogle** [comment]
ping ping

**Comment by toffaletti** [comment]
If you run `setup.py build` before `setup.py install` then the compiler module is included.

**Comment by balopat** [comment]
+1 I'm running into the same issue on 3.5.1

**Comment by davidair** [comment]
I ran into this as well with the latest version available on brew (3.5.1), had to roll back to 2.6 as a workaround:

`brew install protobuf@2.6
brew link protobuf@2.6 --force`
